src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -406,7 +410,12 @@ protected void requestCredentialsUpdate(Context context, Account account) {
             startActivityForResult(updateAccountCredentials, REQUEST_CODE__UPDATE_CREDENTIALS);
 
         } catch (com.owncloud.android.lib.common.accounts.AccountUtils.AccountNotFoundException e) {
-            Toast.makeText(context, R.string.auth_account_does_not_exist, Toast.LENGTH_SHORT).show();
+            Snackbar snackbar = Snackbar.make(
```

Comment:
Same as before, create a new method to show the snackbar, passing the error message as parameter, because the rest of snackbar parameters are the same everytime you show the snackbar.